theseus/src/api/pack/import/atlauncher.rs

```diff
@@ -107,10 +100,35 @@ pub async fn is_valid_atlauncher(instance_folder: PathBuf) -> bool {
     let instance: String =
         io::read_to_string(&instance_folder.join("instance.json"))
             .await
-            .unwrap_or("".to_string());
+            .unwrap_or_else(|e| {
+                // TODO: remove temporary logging in future update
+                if !instance_folder.exists() {
+                    tracing::warn!(
+                        "Instance folder does not exist: {}",
+                        instance_folder.display()
+                    );
+                    return "".to_string();
+                } else {
+                    tracing::warn!(
+                        "Could not read existing instance.json at {}: {}",
+                        instance_folder.display(),
+                        e
+                    )
+                };
+                "".to_string()
+            });
```

Comment:
is this meant to be removed?

Reply:
I meant it to be included so that if the person in disc who's bug I couldn't replicate could have a log when they tried to import their profiles that should be importable.

That being said I do have a guess for what the issue was and fixed it, so I can revert this